src/Cache/_ScsDataClient.php

```diff
@@ -576,7 +576,7 @@ public function dictionaryGetBatch(string $cacheName, string $dictionaryName, ar
             return new CacheDictionaryGetBatchResponseError(new UnknownError($e->getMessage()));
         }
         if ($dictionaryGetBatchResponse->hasFound()) {
-            return new CacheDictionaryGetBatchResponseSuccess($dictionaryGetBatchResponse);
+            return new CacheDictionaryGetBatchResponseSuccess($dictionaryGetBatchResponse, fields: $fields);
```

Comment:
I think you have another PR up that may have conflicts with this one, yes?  Where this class gets renamed to `*DictionaryGetFields*`?  Just calling it out to make sure that we end up with the correct one before the final merge.

Reply:
Yeah this is the PR for that: https://github.com/momentohq/client-sdk-php/pull/64